### ISSUE TITLE
[feature request] compression upon backup

### ISSUE THREAD

**Issue by tangodev** [thundernest/import-export-tools-ng#301]
Hello,

I've been using this extension for quite some time, mainly to perform backups. I'd like to request compression as an extra option (`tar.gz`, `zip`, `others`?).

As a result, instead of a directory `foo`, IET-NG would produce a file, ex. `foo.tar.gz`, `foo.zip` etc..

Rationale:
- backups tend to be in multiple GiBs of size
- text compresses well
- larger, compressed tarballs are easier to handle than directories with tons of small files

**Comment by SylwesterZarebski** [comment]
Yes, it would be very nice to have compressed backups. Especially when you backup to HDD network storage, copying small files (maildir fomat) is very, very slow. Making backup of about 5 GB mails to NAS with HDD disks takes hours.
